### ISSUE TITLE
Bug Report: sorry this isnt a bug report i just had something to ask, wheres the download link???

### ISSUE THREAD

**Issue by ReignBoi** [ash237/vs-Bob-and-Bosip#21]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.

**Comment by JorosTheModderPH** [comment]
Deleted, due to drama.

**Comment by SilverT4** [comment]
the mod was trashed due to controversy. if you're not in the know, there's videos on YouTube. I'll give a bit of a basic summary.

Amor was found to be a pedophile. (Shocking in the FNF community, right? /s) He was also exposed to have drawn NSFW art of his characters and others', some of whom were minors, even when he himself was a minor. Additionally, he had talked to AetherDX (a groomer), although some people believe he was just saying that he hopes Aether can become a better person. Still didn't do anything good for his reputation.

i myself hope amor can become a better person. what he did was unforgivable.

ETA: If you still have the mod on your own computer, there's no shame in playing it on your own computer. It's offline and gives no support to Amor.

**Comment by RagdollClash** [comment]
It would be a better idea to give support to the victims, instead of hoping for the pedophile to better themself. Just saying.

**Comment by SilverT4** [comment]
> It would be a better idea to give support to the victims, instead of hoping for the pedophile to better themself. Just saying.

Agreed. I don't support Amor anymore, nor do I think I can, even if he does better himself.
